Source/WebCore/platform/graphics/avfoundation/SampleBufferDisplayLayer.h

```diff
@@ -62,7 +65,7 @@ class SampleBufferDisplayLayer {
     virtual void updateDisplayMode(bool hideDisplayLayer, bool hideRootLayer) = 0;
 
     virtual void updateAffineTransform(CGAffineTransform) = 0;
-    virtual void updateBoundsAndPosition(CGRect, VideoFrameRotation) = 0;
+    virtual void updateBoundsAndPosition(CGRect, VideoFrameRotation, std::optional<WTF::MachSendRight> = std::nullopt) = 0;
```

Comment:
Should probably be std::optional<WTF::MachSendRight>&&, given MachSendRight has a move constructor

Reply:
> Should probably be std::optionalWTF::MachSendRight&&, given MachSendRight has a move constructor

Good point, will fix!